### PR TITLE
[CARBONDATA-1993] Carbon properties default values fix and correspond…

### PR DESCRIPTION
**Pull request by pawanmalwal**
Carbon properties default values fix and corresponding template and document correction.

 - [X] Any interfaces changed?
 None
 - [X] Any backward compatibility impacted?
 None
 - [X] Document update required?
Updated corresponding template and document
 - [X] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       None

 - [X] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
None

### CODE REVIEW

File: conf/carbon.properties.template [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #Mandatory. Carbon Store path
 carbon.storelocation=hdfs://hacluster/Opt/CarbonStore
 #Base directory for Data files
-carbon.ddl.base.hdfs.url=hdfs://hacluster/opt/data
+carbon.ddl.base.hdfs.url=
```
Comment (KanakaKumar) [inline conf/carbon.properties.template]: If the default value is empty, shall we remove property from template or just comment out and keep ?

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/2596/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/1356/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/2760/

File: conf/carbon.properties.template (hunk 1 above)
Comment (pawanmalwal) [reply conf/carbon.properties.template]: I have commented it for now.

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/2637/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/2643/

**Pull request closed without merging**